src/commands/createTransactionRegisterSecondPassphrase.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import {
+	getStdIn,
+	getPassphrase,
+	getFirstLineFromString,
+} from '../utils/input';
+import { createCommand } from '../utils/helpers';
+import commonOptions from '../utils/options';
+import { createLiskTransaction } from '../utils/liskInstance';
+
+const description = `Creates a register second passphrase transaction for an existing account.
```

Comment:
`Creates a transaction which will register a second passphrase for an existing account if broadcast to the network.`